packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/Sidebar.tsx

```diff
@@ -4,35 +4,38 @@ import { DeviceSelector } from '../DeviceSelector/DeviceSelector';
 import { Navigation } from './Navigation';
 import { AccountsMenu } from 'src/components/wallet/WalletLayout/AccountsMenu/AccountsMenu';
 import { QuickActions } from './QuickActions';
-import { ElevationUp, useElevation } from '@trezor/components';
+import { ElevationUp, ResizableBox, useElevation } from '@trezor/components';
 import { SIDEBAR_WIDTH_NUMERIC } from 'src/constants/suite/layout';
-import { Elevation, mapElevationToBackground, mapElevationToBorder } from '@trezor/theme';
+import { Elevation, mapElevationToBackground, mapElevationToBorder, zIndices } from '@trezor/theme';
 
 const Container = styled.nav<{ $elevation: Elevation }>`
     display: flex;
     flex-direction: column;
     flex: 0 0 auto;
-    width: ${SIDEBAR_WIDTH_NUMERIC}px;
-    resize: horizontal;
-    min-width: 200px;
-    max-width: 400px;
     height: 100%;
     background: ${mapElevationToBackground};
     border-right: 1px solid ${mapElevationToBorder};
-    overflow: auto;
 `;
 
 export const Sidebar = () => {
     const { elevation } = useElevation();
 
     return (
-        <Container $elevation={elevation}>
-            <ElevationUp>
-                <DeviceSelector />
-                <Navigation />
-                <AccountsMenu />
-                <QuickActions />
-            </ElevationUp>
-        </Container>
+        <ResizableBox
+            directions={['right']}
+            width={SIDEBAR_WIDTH_NUMERIC}
+            minWidth={230}
+            maxWidth={400}
+            zIndex={zIndices.draggableComponent}
```

Comment:
I think this (`zIndices.draggableComponent`) could be set as the default value for `zIndex` in `ResizableComponent`